### ISSUE TITLE
Minor source install doc suggestions

### ISSUE THREAD

**Issue by springmeyer** [mapbox/DEPRECATED-mapbox-ios-sdk#222]
I set out to grab sdk from github and do a source install to enable building the sdk inside my custom app. These are my notes. Feel free to close if they are not actionable.
- I got tripped up looking for "Link Binary With Libraries" in the build settings rather than "build phase". For better noob support I would recommend:

``` diff
-Add the following libraries to your app target’s Link Binary With Libraries build phase:
+Next go to your projects "Build Phases" and find the "Link Binary With Libraries" phase:
```
- I also hit unexpected build problems with missing FMDB headers because a source install needs `git submodule update --init` which I had not run. The github README notes this but not the docs at http://mapbox.com/mapbox-ios-sdk/#source, which do not mention following the readme instructions. My suggestion would be to centralize or better cross-reference these docs if they both need to be read for proper setup
- "At a minimum, #import "MapBox.h" and instantiate an RMMapView in your code along with an RMMapBoxSource using -initWithMapID: and an NSString made from your Map ID obtained above." < -- this is usage not install so I would recommend removing this line and instead pointing to whole page on example code directly (so it is obvious)

**Comment by springmeyer** [comment]
also in Step 8 it says "Add the MapBox.bundle product from the SDK to the Copy Bundle Resources build phase of your app target". I stumbled on this because I first clicked the `+` (add items) button at the bottom of the `Copy Bundle Resources` and was unable to find the `MapBox.bundle`. So I gave up and tried the build and hit:

```
2013-03-28 14:03:46.428 mapnik-test[40848:11c03] Opening database at /Users/dane/Library/Application Support/iPhone Simulator/6.1/Applications/88A3F24D-AD9B-460C-A504-20305B3F4FB7/Documents/RMTileCache.db
2013-03-28 14:03:46.431 mapnik-test[40848:11c03] *** Assertion failure in +[RMMapView pathForBundleResourceNamed:ofType:], /Users/dane/projects/ios-testing/mapnik-test/mapbox-ios-sdk/MapView/Map/RMMapView.m:416
2013-03-28 14:03:46.457 mapnik-test[40848:11c03] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Resource bundle not found in application.'
*** First throw call stack:
(0x26a6012 0x22f1e7e 0x26a5e78 0x1d87665 0x526e1f 0x526ca9 0x558a23 0x558905 0x531548 0x526261 0x52693f 0x52683f 0x3267 0x13171c7 0x1317232 0x12663d5 0x126676f 0x1266905 0x126f917 0x123396c 0x123494b 0x1245cb5 0x1246beb 0x1238698 0x3646df9 0x3646ad0 0x261bbf5 0x261b962 0x264cbb6 0x264bf44 0x264be1b 0x123417a 0x1235ffc 0x243d 0x20e5)
libc++abi.dylib: terminate called throwing an exception
```

I realize now for source building this bundle will not exist yet, so I propose the source docs change like:

``` diff
- 8. Add the MapBox.bundle product from the SDK to the Copy Bundle Resources build phase of your app target. This way, the bundle will be copied at build time out of the SDK build products folder and included as a resource in your app product.
+ 8. In the project navigator find the `MapView.xcodeproj` and open up the subdirectory called `Products`. You should see a red `libMapBox.a` and `MapBox.bundle` listed. Now open the `Build Phases -> Copy Bundle Resources` view of your project and drag the `MapBox.bundle` item into the `Copy Bundle Resources` area. This is needed so that bundle will be copied at build time out of the SDK build products folder and included as a resource in your app product. If you miss this step then your app will crash with the error `NSInternalInconsistencyException', reason: 'Resource bundle not found in application.`
```

**Comment by nickdaugherty** [comment]
I too struggled trying to figure out why MapBox.bundle wasn't available, until this issue turned up in Google. +1 for modifying the instructions as @springmeyer suggests.

**Comment by kirscfil** [comment]
Same misunderstanding here, I also suggest @springmeyer 's change.

**Comment by springmeyer** [comment]
bump @incanus

**Comment by incanus** [comment]
I'll take a pass at this soon.

**Comment by boxed** [comment]
I had the same assert fired as above when running on the simulator, but running on a device was fine. Turns out XCode screwed up and didn't properly copy over the bundle even after I hit clean and rebuild. I had to go into ~/Library/Application Support/iPhone Simulator/7.0.3/Applications/[GUID of my app]/ and delete the .app itself before XCode copied the correct one over to the simulator.

I'm writing this here so the next guy who google on this problem can find this solution :)
